source/manual/rules-for-getting-production-access.html.md

```diff
@@ -11,17 +11,53 @@ These rules apply to developers, SREs, and technical architects in the GOV.UK pr
+We have two types of production access:
+
+1. [Production Deploy access](#production-deploy-access)
+2. [Production Admin access](#production-admin-access)
+
+### Production Deploy access
+
+This level of access allows engineers to deploy code but not administer admin related
+systems. It should be granted to both civil servants and contractors as needed.
+
+Access includes:
+
+- Permission to [deploy apps](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#deployment) in Jenkins via the [GOV.UK Production Deploy Github team](https://github.com/orgs/alphagov/teams/gov-uk-production-deploy)
+- Permission to [merge pull requests](https://docs.publishing.service.gov.uk/manual/merge-pr.html#header) in continuously deployed applications
+- Readonly access to logging systems such as Logit, etc.
+- AWS readonly access via the `role_user_user_arns` role in [Staging](https://github.com/alphagov/govuk-aws-data/blob/main/data/infra-security/staging/common.tfvars) and [Production](https://github.com/alphagov/govuk-aws-data/blob/main/data/infra-security/production/common.tfvars)
+- "Normal" access to GOV.UK Signon in Staging and Production (with app permissions granted as needed)
+
+The steps above are outlined in the [GOV.UK Production Deploy template Trello card](https://trello.com/c/S9sex2XU/1391-govuk-production-deploy-access-for-name), which can be copied to
+your team's board and carried out by developers. You can ask 2nd line for help if you have
+any access issues.
+
+#### When you get Production Deploy access
+
+Access should be granted at the discretion of the engineer's tech lead, once the engineer has the required level of security clearance (BPSS). Before approving access, tech leads should ensure that the engineer:
+
+- is aware of our processes and standards around [code review](https://gds-way.cloudapps.digital/manuals/code-review-guidelines.html)
+- understands the responsibilities that [releasing code](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#deployment) brings with it
+- knows how to roll back to an older release if there are any issues
+- knows [how to get help](https://docs.publishing.service.gov.uk/manual/ask-for-help.html) from someone with more access if they need it
+
+### Production Admin access
+
+- Permission to read & write [production](https://github.com/alphagov/govuk-secrets/blob/main/puppet_aws/hieradata/production_credentials.yaml) and [staging](https://github.com/alphagov/govuk-secrets/blob/main/puppet_aws/hieradata/staging_credentials.yaml) [hieradata](https://docs.publishing.service.gov.uk/manual/encrypted-hiera-data.html#what-to-do-when-someone-gets-production-access) in govuk-secrets using GPG
+- Permission to read & write from the [Pass](https://github.com/alphagov/govuk-secrets/tree/main/pass) in govuk-secrets store using [GPG](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/.gpg-id)
```

Comment:
I read this as "the PaaS" - think this would be clearer:

```suggestion
- Permission to read & write to the [password store](https://github.com/alphagov/govuk-secrets/tree/main/pass) in govuk-secrets using [GPG](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/.gpg-id)
```

Reply:
Thanks Chris, I've made this change now.